### DISCUSSION TITLE
How I set ResponseLimit to an endpoint?

### DISCUSSION THREAD

**Discussion by lankinenxyz** [vercel/next.js#53986] [App Router]
Hey,
I'm hitting the 4mb response limit but can't figure out how I can increase it for specific or all endpoints in my App Router project.

**Comment by Nadeera3784** [comment]
Hi @RealLankinen,
[Possible Ways to Fix It](https://nextjs.org/docs/messages/api-routes-response-size-limit#possible-ways-to-fix-it)

**Reply by lankinenxyz** [reply]
Isn't this for Pages Router, not App Router?

**Reply by lankinenxyz** [reply]
I'm getting `Error: `export const config` in /file/route.ts is deprecated. Please change `runtime` property to segment export config. See https://nextjs.org/docs/app/api-reference/file-conventions/route-segment-config` when I try to add
```
export const config = {
  api: {
    responseLimit: false,
  },
};
```

to one `route.ts` file.

**Reply by Capta1nRaj** [reply]
Did u found a fix? Getting same error here too.
